# msg for servo movement

string pos_x
string pos_z